### DISCUSSION TITLE
Add the option to navigation between tabs at the top of the page

### DISCUSSION THREAD

**Discussion by zairro** [GitbookIO/community#655] [Feature Requests]
Hello, I'd like to request the ability to add tabs to the top of the page. This helps with highlighting features with more documentation requirements like APIs stand out in the documentation.

I am including a few screenshots here to provide more clarity on this request.

![CleanShot 2024-07-15 at 12 21 31](https://github.com/user-attachments/assets/78c3bd98-5043-443e-84e9-21e05dc51957)
![CleanShot 2024-07-15 at 12 22 05](https://github.com/user-attachments/assets/a286cf3d-b6f4-41fc-920a-1fa21fd69ae0)

Stripe has an example of this top of navigation here: https://docs.stripe.com/get-started

**Comment by aikoii** [comment]
Hey @zairro! We have a feature that allows you to do this 🙂. You can add links to important parts of your docs to the header of your space/site. 

<img width="462" alt="CleanShot 2024-07-16 at 12 24 44@2x" src="https://github.com/user-attachments/assets/115ad209-bc3f-449d-b2b9-25fa92869dbf">


Here is the link with more info: 
https://docs.gitbook.com/published-documentation/customization/space-customization#header
